build/Shared/SdkAnalysisLevelMinimums.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Shared
+{
+    /// <summary>
+    /// Defines constants that specify the minimum SDK analysis level required to enable new features.
+    /// </summary>
+    internal static class SdkAnalysisLevelMinimums
+    {
+        public const string HttpErrorSdkAnalysisLevelMinimumValue = "9.0.100";
```

Comment:
Can we store the `NuGetVersion` type directly instead of using a string and creating a `NuGetVersion` object every time for the comparison?

Reply:
https://github.com/NuGet/NuGet.Client/pull/5925#discussion_r1678438801

Reply:
It's because this is just a .cs file in the shared folder. It's not in any NuGet project

Reply:
Moved it to NuGet.Versioning, I think it makes things simpler